velox/vector/tests/DecodedVectorTest.cpp

```diff
@@ -25,14 +25,24 @@
 #include "velox/vector/BaseVector.h"
 #include "velox/vector/SelectivityVector.h"
 #include "velox/vector/TypeAliases.h"
+#include "velox/vector/fuzzer/VectorFuzzer.h"
 #include "velox/vector/tests/VectorTestUtils.h"
 #include "velox/vector/tests/utils/VectorTestBase.h"
 
 namespace facebook::velox::test {
 
 class DecodedVectorTest : public testing::Test, public VectorTestBase {
  protected:
-  DecodedVectorTest() : allSelected_(10010), halfSelected_(10010) {
+  VectorFuzzer::Options getFuzzerOptions() {
```

Comment:
do we still need the Fuzzer?
Can you clean up the related code to simplify the review?

Reply:
Done.